packages/frint/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,72 @@
+# Contributor Covenant Code of Conduct
```

Comment:
You can move this file to the root of the repository.

Reply:
thanks @markvincze and @fahad19 I will update it in a while :)